modules/fedimint-mint-client/src/lib.rs

```diff
@@ -1163,6 +1188,16 @@ impl MintClientModule {
     ///
     /// Static to help re-use in other places, that don't have a whole [`Self`]
     /// available
+    ///
+    /// # E-Cash Note Creation
+    ///
+    /// When creating an e-cash note, the `MintClientModule` first derives the
+    /// blinding and spend keys from the `DerivableSecret`. It then creates a
+    /// `NoteIssuanceRequest` containing the blinded spend key and sends it to
+    /// the mint server. The mint server signs the blinded spend key and
```

Comment:
`containing the blinded spend key` should probably be `containing the blinded spend public key`.